spicy/src/compiler/codegen/parser-builder.cc

```diff
@@ -409,7 +409,10 @@ struct ProductionVisitor
         const auto& field = meta.field();
         assert(field); // Must only be called if we have a field.
 
-        if ( field && is_field_owner && ! meta.container() ) {
+        if ( ! is_field_owner )
```

Comment:
This disables all attribute handling in this function if `! is_field_owner`? Is that intentional? Since this function does so much it might be easier to follow the code if we'd avoid the early return and instead wrap handling not applicable in e.g., `if ( is_field_owner) ...` or similar.

Reply:
Yes, on purpose, but let me make it more clear.